fortran/plumed_f08.f90.template

```diff
@@ -181,21 +181,10 @@ module plumed_f08_module
 
 #: for t in types
 #: for k in kinds[t]
-  interface
```

Comment:
I agree with removing this. It was there since it is needed in gfortran 4 (and I originally copied this from the plumed.f90 interface, compatible with gfortran 4), but it can be removed here